python/paddle/fluid/layers/nn.py

```diff
@@ -10561,3 +10562,50 @@ def tree_conv(nodes_vector,
     else:
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+def cvm(input, cvm, use_cvm=True):
+    """
+    **cvm layers**
+
+    This layer accepts a tensor named input which is ID after embedded and lod level is 1 ,
+         cvm is a show_click info and returns the cvm output.
+
+    Args:
+        input (Variable):  a 2-D LodTensor with shape [N x D], where N is the
+                                batch size, D is the embedding dim. 
+                                lod level = 1.
+        cvm (Variable):    a 2-D Tensor with shape [N x 2], where N is the batch size, 2 is show and click.
+        use_cvm  (bool):  use cvm or not. if use cvm, the output dim is the same as input
```

Comment:
do you think it's confusing? how can cvm layer not using cvm?

Reply:
yeah，it's a little confusing. cvm op is a  customized op, which input is a sequence had embedd_with_cvm default, so we need a op named cvm to decided whever use it or not.